dash_auth/plotly_auth.py

```diff
@@ -21,6 +22,19 @@
 from . import api_requests
 
 
+deprecation_notice = '''
+PlotlyAuth is being deprecated.
+If your app is still using dash-deployment-server < 2.6,
```

Comment:
minor, but the product is called Dash Deployment Server (it's not a Python package).